adapter/main.go

```diff
@@ -111,6 +112,11 @@ func main() {
 	printVersion()
 
 	cmd := &Adapter{}
+
+	cmd.OpenAPIConfig = genericapiserver.DefaultOpenAPIConfig(generatedopenapi.GetOpenAPIDefinitions, openapinamer.NewDefinitionNamer(scheme.Scheme))
+	cmd.OpenAPIConfig.Info.Title = "keda-adapter"
+	cmd.OpenAPIConfig.Info.Version = "1.0.0"
```

Comment:
Should this version colerate with KEDA version or with OpenAPI spec version?

Reply:
This is "application api version", not application version or spec version. Please see here: https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md#infoObject

Reply:
okay, thanks!